app/styles/partials/footer.scss

```diff
@@ -33,3 +33,68 @@ footer {
     right: 0 !important;
   }
 }
+
+
+@media screen and (min-width : 1200px) {
+    #main-container {
```

Comment:
@sumedh123 the code needs to work on any page with side rails ... Have you css selectors like that